src/workers/postUpdated.ts

```diff
@@ -485,7 +495,6 @@ const fixData = async ({
       showOnFeed: !data?.order,
       flags: {
         private: privacy,
-        visible: becomesVisible,
```

Comment:
Removed it from `fixData` because we need to know if current post exist to properly deduce current visible status.